requirements.txt

```diff
@@ -39,4 +39,5 @@ apscheduler
 git+https://github.com/openimis/django-apscheduler.git#egg=django-apscheduler
 django-simple-history
 django-dirtyfields==1.4.1
-daphne==3.0.1
\ No newline at end of file
+daphne==3.0.1
+GitPython
```

Comment:
Can you attach version of package to requirement?

Reply:
Ok. I need to check current version of this and I will put version too.